### DISCUSSION TITLE
Social Media

### DISCUSSION THREAD

**Discussion by RJGhugo** [opentofu/opentofu#566] [General]
I do  not have a Twitter Account!

Do you have a Fediverse/Mastodon Account that I could follow?

**Comment by davidjeddy** [comment]
I do have Twitter/X, but no longer have a desire to use the service; I doubt I am the only one. +1 to any and all other options.

**Comment by RJGhugo** [comment]
It is more about the project news. There are a lot of OpenSource project posting in the Fediverse/Mastadon.
